core/src/test/java/com/google/zxing/datamatrix/encoder/HighLevelEncodeTestCase.java

```diff
@@ -500,7 +500,7 @@ public void testECIs() {
         "\u062C\u064E\u0651\u0627\u0635 (\u02BE\u0101\u1E63) \"pear\", suggested to have originated from Hebrew " +
         "\u05D0\u05B7\u05D2\u05B8\u05BC\u05E1 (ag\u00E1s)"));
     assertEquals("239 209 151 206 214 92 122 140 35 158 144 162 52 205 55 171 137 23 67 206 218 175 147 113 15 254" +
-        " 116 33 241 25 231 186 14 212 64 253 151 252 159 33 41 241 27 231 83 171 53 209 35 25 134 6 42 33 35 239 184" +
+        " 116 33 241 9 231 186 14 206 64 248 144 252 159 33 41 241 27 231 83 171 53 209 35 25 134 6 42 33 35 239 184" +
```

Comment:
Ah OK, this change is slightly concerning. Do you know why it happens? I can probably go figure it out. Presumably because some new charset was chosen

Reply:
Correct 241 + 25 = ECI 24 (Cp1256), and 241 + 9 = ECI 8 (ISO-8859-6). This test was added after ECI 8 was commented out, so ECI 24 was the next-best choice at the time. By adding ECI 8 back in, this test now prefers it over ECI 24.